tools/shell-commands/src/main/java/org/apache/unomi/shell/migration/utils/MigrationUtils.java

```diff
@@ -281,6 +283,23 @@ public static void waitForYellowStatus(CloseableHttpClient httpClient, String es
 
     }
 
+    public static void waitForTaskToFinish(CloseableHttpClient httpClient, String esAddress, String taskId, MigrationContext migrationContext) throws IOException {
+        while (true) {
+            final JSONObject status = new JSONObject(
+                    HttpUtils.executeGetRequest(httpClient, esAddress + "/_tasks/" + taskId + "?wait_for_completion=true&timeout=10s",
+                            null));
+            if (status.has("completed") && status.getBoolean("completed")) {
+                migrationContext.printMessage("Task is completed");
+                break;
+            }
+            if (status.has("error")) {
+                final JSONObject error = status.getJSONObject("error");
+                throw new IOException("Task error: " + error.getString("type") + " - " + error.getString("reason"));
+            }
+            migrationContext.printMessage("Waiting for Task " + taskId + " to complete");
```

Comment:
Don't we have a risk to have too much logs here. 
According to the time of the reindex, we might have too much log displayed.

Reply:
Yes I know but migration is not something regular, and knowing that something is running may be useful. What should be the frequency for that log ? 5mn ?

Reply:
Every 20-30 secondes is a good option I think